synpurge/cli.py

```diff
@@ -24,7 +35,7 @@ def version():
 
 @cmd
 def check(path: "configuration file",
-          verbose: "display configuration" = False):
```

Comment:
This change is unrelated to the purpose of the patch. Please leave this out from the PR.